app/main.py

```diff
@@ -1,7 +1,26 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+
+
+def get_weather(city: str, key: str) -> None:
+    try:
+        response = requests.get(f"{URL}key={API_KEY}&q={CITY}&aqi=no")
```

Comment:
The function `get_weather` is defined to take `city` and `key` as parameters, but the request URL is using the global variables `API_KEY` and `CITY` instead. Consider using the function parameters `city` and `key` in the request URL to make the function more flexible and reusable.